### ISSUE TITLE
can not import default in .vue files

### ISSUE THREAD

**Issue by fenivana** [nuxt/nuxt.js#63]
If importing the default object of `.js` file in a `.vue` file, the default object will be wrapped.

foo.js
```js
export default {
  bar: 123
};
```

some.vue
```js
import foo from './foo';
// need to use foo.default to access the default object
console.log(foo.default.bar);
```

I think the problem is because `vue-loader` did not config the babel option: `modules: false` at https://github.com/nuxt/nuxt.js/blob/master/lib/build/webpack/vue-loader.config.js#L7
```js
  let config = {
    postcss: this.options.build.postcss,
    loaders: {
      'js': 'babel-loader?presets[]=es2015&presets[]=stage-2',
      'less': 'vue-style-loader!css-loader!less-loader',
      'sass': 'vue-style-loader!css-loader!sass-loader?indentedSyntax',
      'scss': 'vue-style-loader!css-loader!sass-loader',
      'stylus': 'vue-style-loader!css-loader!stylus-loader',
      'styl': 'vue-style-loader!css-loader!stylus-loader'
    }
  }
```
I wonder why set the babel options here rather than use the .babelrc ?

**Comment by Atinux** [comment]
Hi @fenivana 

Thanks for spotting it, it will be fixed in the next release 👍

**Comment by Atinux** [comment]
Fixed in [`0.9.0`](https://github.com/nuxt/nuxt.js/releases/tag/v0.9.0)
